data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2275,10 +2275,28 @@ spec:
                       NetworkProjectID specifies which project the network and subnets
                       exist in when they are not in the main ProjectID.
                     type: string
+                  privateDNSZone:
+                    description: PrivateDNSZone Technology Preview. The PrivateDNSZone
+                      should be provided as "projects/{project-id}/managedZones/{managed-zone-name}".
+                      When the PrivateDNSZone is empty, the DNS zones will be created
+                      and managed in the Service Project (ProjectID). When "None",
```

Comment:
We are checking if privateDNSZone is set to gcp.DNSZoneDisabled which is defined by the constant 
DNSZoneDisabled string = "Unmanaged".

So, should the description here be updated to reflect that?